### PR TITLE
chore(bindings): Use tracing-android instead of android-logger

### PR DESCRIPTION
**Pull request by poljar**

### CODE REVIEW

**Review by jplatte — COMMENTED** [review]
But why remove `android_logger` already without a replacement? I think we should use it for now on both android and ios (tracing will collect `log` events), and then replace it with a natively tracing-enabled solution afterwards.

**Comment by poljar** [comment]
> But why remove `android_logger` already without a replacement? I think we should use it for now on both android and ios (tracing will collect `log` events), and then replace it with a natively tracing-enabled solution afterwards.

You mean `log-panics`? Yeah, I assumed that #1498 would be the way we would go.

File: bindings/matrix-sdk-ffi/Cargo.toml [hunk 1]
```diff
@@ -40,12 +41,11 @@ zeroize = { workspace = true }
 
 
 [target.'cfg(target_os = "android")'.dependencies]
-tracing = { version = "0.1.29", default-features = false, features = ["log"] }
-android_logger = "0.11"
 log-panics = { version = "2", features = ["with-backtrace"]}
```
Comment (jplatte) [inline bindings/matrix-sdk-ffi/Cargo.toml:44]: Still android-specific? Do you want to change that later when replacing it?

**Review by jplatte — APPROVED** [review]

File: bindings/matrix-sdk-ffi/Cargo.toml (hunk 1 above)
Comment (poljar) [reply bindings/matrix-sdk-ffi/Cargo.toml:44]: Yep.

**Pull request merged**